share/picongpu/examples/TransitionRadiation/include/picongpu/param/transitionRadiation.param

```diff
@@ -0,0 +1,264 @@
+    }
+
+    //! number of frequency values to compute in the linear frequency [unitless]
+    constexpr unsigned int nOmega = 512;
+
+} // namespace linearFrequencies
+
+namespace logFrequencies
+{
+    namespace SI
+    {
+        //! mimimum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMin = 1.0e13;
+        //! maximum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMax = 1.0e17;
+    }
+
+    //! number of frequency values to compute in the logarithmic frequency [unitless]
+    constexpr unsigned int nOmega = 256;
+
+} // namespace logFrequencies
+
+
+namespace listFrequencies
+{
+    //! path to text file with frequencies
+    constexpr char listLocation[] = "/path/to/frequency_list";
+    //! number of frequency values to compute if frequencies are given in a file [unitless]
+    constexpr unsigned int nOmega = 512;
+
+} // namespace listFrequencies
+
+
+    /** selected mode of frequency scaling:
+     *
+     * options:
+     * - linearFrequencies
+     * - logFrequencies
+     * - listFrequencies
+     */
+    namespace frequencies = logFrequencies;
+
+    ///////////////////////////////////////////////////
+
+
+    /** correct treatment of coherent radiation from macro particles
+     *
+     * These formfactors are the same as in the radiation plugin!
+     * Choose different form factors in order to consider different particle shapes for radiation
+     *  - ::picongpu::plugins::radiation::radFormFactor_CIC_3D ... CIC charge distribution
+     *  - ::picongpu::plugins::radiation::radFormFactor_TSC_3D ... TSC charge distribution
+     *  - ::picongpu::plugins::radiation::radFormFactor_PCS_3D ... PCS charge distribution
+     *  - ::picongpu::plugins::radiation::radFormFactor_CIC_1Dy ... only CIC charge distribution in y
+     *  - ::picongpu::plugins::radiation::radFormFactor_Gauss_spherical ... symmetric Gauss charge distribution
+     *  - ::picongpu::plugins::radiation::radFormFactor_Gauss_cell ... Gauss charge distribution according to cell size
+     *  - ::picongpu::plugins::radiation::radFormFactor_incoherent ... only incoherent radiation
+     *  - ::picongpu::plugins::radiation::radFormFactor_coherent ... only coherent radiation
+     */
+    namespace macroParticleFormFactor = ::picongpu::plugins::radiation::radFormFactor_Gauss_spherical;
+
+    ///////////////////////////////////////////////////////////
+
+    namespace parameters
+    {
+        // number of observation directions
+        constexpr unsigned int nPhi = 128;
+        constexpr unsigned int nTheta = 128;
+        constexpr unsigned int nObserver = nPhi * nTheta;
+
+        // theta goes from 0 to pi
+        constexpr float_64 thetaMin = 0.0;
+        constexpr float_64 thetaMax = picongpu::PI;
+
+        // phi goes from 0 to 2*pi
+        constexpr float_64 phiMin = 0.0;
+        constexpr float_64 phiMax = 2 * picongpu::PI;
```

Comment:
This looks like the default `transitionRadiation.param`. If this is the case, you could avoid adding it - which however would be strange since this is the example for OTR.